i18n/translations/fr/form-builder.json

```diff
@@ -1301,5 +1301,18 @@
   },
   "systemHealthCheck": {
     "title": "Santé du système"
+  },
+  "saveAndResume": {
```

Comment:
@anikbrazeau 

FYI - this is merged but letting you know strings are in place for the settings toggle under form management.